package.json

```diff
@@ -35,13 +36,17 @@
   },
   "homepage": "https://github.com/andreizanik/cookies-next#readme",
   "dependencies": {
-    "cookie": "^0.7.0",
-    "@types/cookie": "^0.6.0"
+    "@types/cookie": "^0.6.0",
+    "cookie": "^0.7.0"
```

Comment:
Hello! 
Thanks for your job! 
Could you please try to update cookie package version to 1.0.1? 
I don't know if it works well, but it's the latest version and I would like to update to it
I suggest just trying it out. If there are no problems, we'll update it.

Reply:
sure! I'll try it and let you know if it works out!

Reply:
Using `cookie@1.0.1` internally with no problems after adjusting the code according to its breaking changes in v1. Marking this conversation as resolved :)

cc @andreizanik